src/TokenFuzzChecks.sol

```diff
@@ -109,6 +111,29 @@ contract TokenFuzzChecks is TokenChecks {
         TokenLike(_token).burn(to, burnAmount);
     }
 
+    function checkBurnInsufficientAllowanceFuzz(
+        address _token,
+        string memory _contractName,
+        address to,
```

Comment:
should be `from`

Reply:
It's a bit weird in the transferFrom one we are not also randomizing the `from`.

Reply:
Also, the check is built a bit different than the transferFrom, I guess it is fine, but not sure if would be better to keep them the same.